test/OrchardCore.Tests/Localization/LocalizationServiceTests.cs

```diff
@@ -0,0 +1,17 @@
+using OrchardCore.Localization;
+
+namespace OrchardCore.Tests.Localization;
+
+public class LocalizationService
+{
+    [Fact]
+    public void GetAllCulturesAndAliases_ShouldContainsChineseCultures()
+    {
+        // Act
+        var cultures = ILocalizationService.GetAllCulturesAndAliases();
+
+        // Assert
+        Assert.Contains(cultures, c => c.Name == "zh-CN");
```

Comment:
Would be good here to check if `zh-CN` is there **and only once**, so that we can confirm that our current `Union()` prevents duplicates.